modules/developer-cli-odo-creating-services-from-yaml-files.adoc

```diff
@@ -14,14 +14,14 @@ Printing and modifying the YAML used to start a service
 +
 [source,terminal]
 ----
-$ odo service create <operator-name> --crd <cr-name> --dry-run
+$ odo service create <operator-name>  --dry-run
```

Comment:
NIT - extra space between` <operator-name>` and --dry-run.

Reply:
@lamek done! Thank you.